.husky/pre-commit

```diff
@@ -1 +1,2 @@
-# npm run lint:fix TODO: uncomment this when ts errors are all fixed
+npx lint-staged
+npm run test
```

Comment:
I don't think we should run tests on commit. This works for now while we have low test coverage, but eventually these tests will take too long to run for how frequently we commit. Our pipeline will protect us from merging things that break tests.

Reply:
Removed